src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Information {
+    private String color = new ColorSupplier().getRandomColor();
```

Comment:
we should pass color in constructor while creating new object, 
```suggestion
    private String color;
```